stdlib/effect.mli

```diff
@@ -16,13 +16,21 @@
 
     @since 5.0 *)
 
-type _ t = ..
+type 'a t = 'a eff = ..
 (** The type of effects. *)
 
+exception Unhandled_effect : 'a eff -> exn
+(** Exception raised when an effect is performed and there is no matching
+    handler for that effect. The argument is the unhandled effect. *)
+
+exception Continuation_already_taken
+(** Exception raised when a continuation is resumed with a [continue] or
```

Comment:
Please link to the functions, i.e. `{!continue}` or `{!discontinue}`.

Reply:
There is a `Deep.continue` and `Shallow.continue` and similarly two `discontinue`s. I wasn't sure how to word it succinctly but also link to the right ones.

Reply:
or `… more than once.`.